.eslintrc

```diff
@@ -0,0 +1,44 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["@appfolio/eslint-config-appfolio-react", "prettier"],
+  "plugins": ["no-only-tests", "react-hooks", "@typescript-eslint"],
+  "rules": {
+    "curly": ["error", "all"],
+    "react/jsx-props-no-spreading": "off",
+    "react/static-property-placement": "off",
+    "react/destructuring-assignment": "off",
+    "@typescript-eslint/consistent-type-imports": "error",
+    "@typescript-eslint/no-unused-vars": "warn", // TURN THIS BACK ON
+    "function-paren-newline": "off",
+    "import/first": "error",
+    "import/newline-after-import": "warn",
+    "import/order": ["warn", { "alphabetize": { "order": "asc" } }],
+    "max-len": "off",
+    "no-confusing-arrow": "off",
+    "no-only-tests/no-only-tests": "error",
+    "no-nested-ternary": "off",
+    "no-unused-vars": "off",
+    "react/default-props-match-prop-types": ["error", { "allowRequiredDefaults": true }],
+    "react/no-array-index-key": "warn", // TODO: Make this an error when warnings are addressed
+    "react/no-find-dom-node": "warn", // TODO: Make this an error when warnings are addressed
```

Comment:
Are these TODOs still applicable?

Reply:
I suppose not! They have been addressed in code now (either through code changes, or through `eslint-disable` comments with an explanation.